tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestSupplementalInfo.java

```diff
@@ -1587,9 +1585,14 @@ public void TestSupplementalCurrency() {
                 cldrCountries.add(x.getFirst());
             }
             if (!isoCountries.equals(cldrCountries)) {
-                if (!logKnownIssue(
-                        "cldrbug:10765", "Missing codes compared to ISO: " + missing.toString())) {
-
+                // TODO 17397: remove isKnownIssue and the if around errln when the logknown issue
+                // goes away.
+                final boolean skipKnownIssue =
+                        currency.equals("ANG")
+                                && isoCountries.isEmpty()
+                                && cldrCountries.equals(Set.of("CW", "SX"))
+                                && logKnownIssue("CLDR-17397", "Mismatched codes " + cldrCountries);
+                if (!skipKnownIssue) {
```

Comment:
this is now a very specific logKnownIssue.  It won't mask future issues, even if we somehow don't work on CLDR-17397